src-electron/generator/helper-zcl.js

```diff
@@ -2209,6 +2209,78 @@ async function if_mfg_specific_cluster(clusterId, options) {
   }
 }
 
+/**
+ *
+ * @param {*} value
+ * @param {*} size
+ * @returns The negative hex value for a given positive hex value padded with
+ * the given size.
+ */
+async function convert_float_to_hex(value, size) {
+  let res = ''
+  const arrayBuffer = new ArrayBuffer(size)
+  const dataView = new DataView(arrayBuffer)
+  dataView.setFloat32(0, value, false)
+  for (let i = 0; i < size; i++) {
+    let t = dataView.getUint8(i).toString(16)
+    res += t == 0 ? '00' : t
+  }
+  return '0x' + res
+}
+
+/**
+ *
+ * @param {*} value
+ * @param {*} size
+ * @returns the negative hex value for a given positive hex value padded with
+ * the given size.
+ */
+async function get_negative_hex_value(value, size) {
```

Comment:
I thought we generally don't use `get_` for our helpers. We mostly do `as_` or something?

Reply:
fixed it